lms/templates/public_video.html

```diff
@@ -11,6 +11,12 @@
 <meta data-rh="true" property="og:title" content="${social_sharing_metadata['video_title']}">
 <meta data-rh="true" property="og:description" content="${social_sharing_metadata['video_description']}">
 <meta data-rh="true" property="og:image" content="${social_sharing_metadata['video_thumbnail']}">
+<meta data-rh="true" property="og:url" content="${social_sharing_metadata['video_url']}">
+<meta data-rh="true" property="og:video" content="${social_sharing_metadata['video_embed_url']}">
+<meta data-rh="true" property="og:video:url" content="${social_sharing_metadata['video_embed_url']}">
```

Comment:
`Youtube` use `og:video:url` and `Netflix` use `og:video`. I just want to cover all of the cases.